sdk/inc/azure/iot/az_iot_provisioning_client.h

```diff
@@ -477,14 +477,15 @@ az_iot_provisioning_client_payload_options_default();
  * @param[in] client The #az_iot_provisioning_client to use for this call.
  * @param[in] custom_payload_property __[nullable]__ Custom JSON to be added to this payload.
  * Can be `NULL`.
- * @param[in] options __[nullable]__ Reserved field for future options to this function.  Must be
- * `NULL`.
+ * @param[in] options __[nullable]__ A reference to an #az_iot_provisioning_client_payload_options
+ * structure. If `NULL` is passed, the function will use the default options. If using custom
```

Comment:
```suggestion
 * structure. If `NULL` is passed, the provisioning client will use the default options. If using custom
```

Reply:
This parallels the description used for the az_iot_hub_client.

Reply:
Also, the description at line 88 for the provisining_client init does not parallel the iot hub client init in its options description if NULL... is it relevant to update that as well?

Reply:
Re "function": Initially I was thinking that the function is using payload options (and not client options) but this is really using the same provisioning client object so your proposal makes more sense. Will change.

Thanks for noticing the DPS client: will change.